src/main/kotlin/uk/gov/gdx/datashare/models/EventNotification.kt

```diff
@@ -22,8 +23,15 @@ data class EventNotification(
     example = "DEATH_NOTIFICATION",
   )
   val eventType: EventType,
-  @Schema(description = "ID from the source of the notification", required = true, example = "999999901")
-  val sourceId: String,
+
```

Comment:
We don't have this whitespace between other properties, I think we should be consistent but I don't care which way :)

Reply:
done